playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -363,6 +363,18 @@
     - service_variant_config
     - deploy
 
+# generate any declared database cache
+# tables (will make in default db)
+- name: create database cache tables
+  django_manage:
+    app_path: "{{ edxapp_code_dir }}"
+    command: "lms createcachetable"
+    settings: "{{ EDXAPP_SETTINGS }}"
+    virtualenv: "{{ edxapp_venv_dir }}"
+  environment: "{{ edxapp_environment }}"
+  ignore_errors: true
```

Comment:
I don't think we should ignore the errors if you think this command is necessary.

```suggestion
```

Reply:
Yeah, true — I can't remember why I had put that in the Ginkgo version.